test/tests/providers/capi/capa-mimir/platform/atlas/alerting-rules/mimir.rules.test.yml

```diff
@@ -171,4 +170,157 @@ tests:
               opsrecipe: "mimir/"
       - alertname: MimirRestartingTooOften
         eval_time: 140m  # After 140m minutes, all should be back to normal
+  # Test for MimirIngesterNeedsToBeScaledUp alert
+  - interval: 1m
+    input_series:
+      - series: 'sum by (cluster_id, installation, namespace, pipeline, provider) (container_memory_working_set_bytes{container="ingester", namespace="mimir", cluster_type="management_cluster"}'
+        values: "15+0x20 23+0x40 16+0x140 23+0x40 15+0x60" # mimir-ingester real memory usage gradually increases until it goes beyond 90% of the memory requests.
+      - series: 'sum by(cluster_id, installation, namespace, pipeline, provider) (kube_pod_container_resource_requests{container="ingester", namespace="mimir", unit="byte", cluster_type="management_cluster"}'
+        values: "24+0x300"                                 # mimir-ingester memory requests stay the same for the entire duration of the test.
+      - series: 'sum(rate(container_cpu_usage_seconds_total{container="ingester", namespace="mimir", cluster_type="management_cluster"}[5m])) by (cluster_id, installation, namespace, pipeline, provider)'
```

Comment:
Oh you should not use sums and rates here but raw metrics

Reply:
Ughhh what a nightmare, this makes the whole thing even worse to write... But I guess I don't really have a choice :shrug: